### DISCUSSION TITLE
Correct Solution for STR WF26

### DISCUSSION THREAD

**Discussion by janczmok1970** [PricelessToolkit/ESPBell-MAX#2] [Q&A] [answered]
Hi,
i'd like to know if my Doorbell is supported. I'd assume yes, as the Nuki Opener seems to work with it.

![f10342fd9626f658ce9b0e15bc25009b77e87d9c](https://github.com/PricelessToolkit/ESPBell-MAX/assets/139459100/d43869c8-8f0b-4013-8367-ee80310e84b4)
![IMG_0081](https://github.com/PricelessToolkit/ESPBell-MAX/assets/139459100/6d8f3e0c-4bdf-404b-b58a-b0f34f0df3c6)

Thanks for any info.

**Comment by PricelessToolkit** [comment, marked as answer]
Yes, the analog intercom will work, because it has GND, OPEN, and RING cables separately.  why do you want to swap Nuki with ESPbell ?

**Comment by janczmok1970** [comment]
i dont have a nuki, just did research what might work the str wf26 …and i found a nuki thread to be very insightful…hence my question; so i will order one espbell max…Von meinem iPhone gesendetAm 17.12.2023 um 15:59 schrieb PricelessToolkit ***@***.***>:﻿
Yes, the analog intercom will work, because it has GND, OPEN, and RING cables separately.  why do you want to swap Nuki with ESPbell ?

**Comment by janczmok1970** [comment]
So as far i know on the STR26 i have 5 PINS:

1- SPK
2- OPN
3- GND
4- RNG Main Door
5- RNG House Door

So i'd assume the wiring is to the ESPbell MAX:
STR26 - ESPBELL MAX
3 to DB1+R1A
4 to DB2
2 to R1B

Am i correct with my assumption ?

**Reply by PricelessToolkit** [reply]
yes but, you have 2 RING "doorbells" I assume only the Ring of the main Door will work, or is it connected together?

**Reply by PricelessToolkit** [reply]
if only one works, you can try adding two diodes "but I'm not 100 percent sure it will work", tell me when ordering, I will include them in the package.

**Reply by PricelessToolkit** [reply]
like 

4 - diode - DB2
5 - diode - DB2

**Reply by janczmok1970** [reply]
i will check with the diodes later as well.

**Comment by janczmok1970** [comment]
Nope. Main door is the one which rings if somebody at the ground floor ringing my doorbell. the other is the bell at my apartment door. i only want the main ground floor to ring and open.

**Reply by janczmok1970** [reply]
I give it a try later today

**Reply by PricelessToolkit** [reply]
ok, then nothing additional is required
